config/tests/samples/create/samples_test.go

```diff
@@ -301,6 +302,10 @@ func setup() {
 			logging.Fatal(err, "error starting manager")
 		}
 	}()
+
+	if couldNotSyncACache := mgr.GetCache().WaitForCacheSync(ctx); couldNotSyncACache {
```

Comment:
This _could_ be it, but I'm not sure.  I think cached reads are blocked until the caches are synced (I can't remember if the controllers _also_ wait, I think they do not).  If it's the pause tests though, it should be in harness, not here

Reply:
yeah I realized after I ran a test locally that I had the wrong mgr 🤦🏼